mod/crepair.php

```diff
@@ -64,20 +51,22 @@ function crepair_post(App $a)
 	$remote_self = defaults($_POST, 'remote_self', false);
 	$nurl        = Strings::normaliseLink($url);
 
-	$r = q("UPDATE `contact` SET `name` = '%s', `nick` = '%s', `url` = '%s', `nurl` = '%s', `request` = '%s', `confirm` = '%s', `notify` = '%s', `poll` = '%s', `attag` = '%s' , `remote_self` = %d
-		WHERE `id` = %d AND `uid` = %d",
-		DBA::escape($name),
-		DBA::escape($nick),
-		DBA::escape($url),
-		DBA::escape($nurl),
-		DBA::escape($request),
-		DBA::escape($confirm),
-		DBA::escape($notify),
-		DBA::escape($poll),
-		DBA::escape($attag),
-		intval($remote_self),
-		intval($contact['id']),
-		local_user()
+	$r = DBA::update(
```

Comment:
It seems to be a `Contact::update()` method.

Reply:
This method doesn't exist yet, I'd prefer to create it when we have a specific behavior for it.

Reply:
OK, I thought it's better to differ between repository/model- and direct DB-calls.

Reply:
It probably is, but it isn't in the scope of this already heavy PR.

Reply:
OK